pandas/tests/groupby/test_groupby.py

```diff
@@ -2134,3 +2134,25 @@ def test_groupby_column_index_name_lost_fill_funcs(func):
     result = getattr(df_grouped, func)().columns
     expected = pd.Index(["a", "b"], name="idx")
     tm.assert_index_equal(result, expected)
+
+
+@pytest.mark.parametrize("func", ["max", "min"])
+def test_groupby_rolling_index_changed(func):
```

Comment:
Put tests near other similar tests - in this case the groupby/rolling tests are in window/test_grouper.

Reply:
Thanks very much. I searched for them yesterday, but it was already late :)